mrr/src/main/java/gov/cdc/dex/mrr/VocabClient.kt

```diff
@@ -45,7 +46,8 @@ class VocabClient {
                 valuesetConcepts.let {
                     for (i in valuesetConcepts) {
                         val vkey = i.conceptCode
-                        jedis.hset(vocabKey, vkey, i.toString())
+
+                        jedis.hset(vocabKey, vkey, Gson().toJson(i))

```

Comment:
to aleviate the GC having to work so hard, can you create a Gson instance outside the for-loop and reuse it inside it?